specification/semantic_conventions.md

```diff
@@ -76,3 +82,96 @@ Web RUM library MUST add the following attributes to all spans:
 | ----                     | ----   | -----------                                                                                                                    |
 | `location.href`          | string | Value of [`location.href`](https://developer.mozilla.org/en-US/docs/Web/API/Location/href) at the moment of creating the span. |
 | `splunk.script_instance` | string | A 64bit identifier, unique to every instance of the `SplunkRum` library.                                                       |
+
+## Profiling ResourceLogs
+
+**Status**: [Experimental](../README.md#versioning-and-status-of-the-specification)
+
+`ResourceLogs` is the upstream protobuf data type. It MUST be populated with
+the attributes from the OpenTelemetry resource.
+
+### InstrumentationLibraryLogs
+
+Each `ResourceLogs` instance has an instance of `InstrumenationLibraryLogs`.
+For each `InstrumentationLibraryLogs` instance:
+
+- `name` - MUST be set to `otel.profiling`
+- `version` - MUST be set to `0.1.0`
```

Comment:
Why spec prescribes what looks like implementation version?

Reply:
The intent is to allow ingest/backend to be able to handle future changes to log body contents. So if `0.1.0` is just blob of text containing a stack trace but we version `0.2.0` to contains a more structured format (pprof or similar), the version could be used to detect this. 

Your question does make me wonder if `schemaUrl` isn't better suited to this purpose?

Reply:
This is certainly not an intended use of `InstrumentationLibrary.version`. Schemas are probably better for this.